src/pytypes/onefuzztypes/enums.py

```diff
@@ -357,3 +361,16 @@ class GithubIssueSearchMatch(Enum):
 class TaskDebugFlag(Enum):
     keep_node_on_failure = "keep_node_on_failure"
     keep_node_on_completion = "keep_node_on_completion"
+
+
+class UserFieldOperation(Enum):
+    add = "add"
+    replace = "replace"
+
+
+class UserFieldType(Enum):
```

Comment:
Per @chkeita, "might be worth having a guid type"

Reply:
Given these are prompts to users, the only GUIDs for tasks & jobs are things we specifically don't want them to specify.  Namely, job_id and task_id.

As such, I am going to hold off on implementing those for now.